.coveragerc

```diff
@@ -0,0 +1,11 @@
+# .coveragerc to control coverage.py
+
+[report]
+exclude_lines = 
+    if __name__ == '__main__':
+
+[run]
+omit = 
+    sheet_to_triples/debug.py
```

Comment:
Though debug is _for_ debugging, it is live code and exposed as a flag, so shouldn't really be ignored?